frontend/sw360-portlet/src/main/resources/META-INF/resources/html/components/splitComponent.jsp

```diff
@@ -73,6 +73,27 @@
 <%@ include file="/html/utils/includes/requirejs.jspf" %>
 <script>
     require(['jquery', 'bridges/datatables', 'modules/mergeWizard' ], function($, datatables, wizard) {
+        let releaseTooltipFormatter = function(release) {
+            if (!release) {
+                return '';
+            }
+            return 'Source Code Download Url - ' + (release.sourceCodeDownloadurl || '') + '\n' + 'Binary Download Url - ' + (release.binaryDownloadurl || '');
```

Comment:
Is it possible to read `Source Code Download Url` and `Binary Download Url` from language properties file.